### DISCUSSION TITLE
Model-Viewer v1.10.1 & v1.10.0 doesn't load on IOS 13 & 12 safari

### DISCUSSION THREAD

**Discussion by allancozart** [google/model-viewer#3181] [Q&A]
While testing in Browserstack, I noticed that model-viewer  v1.10.1 & v1.10.0 doesn't load in IOS safari versions under 14.

also
any version of 1.9 only displays a black model.

**Comment by elalish** [comment]
Yes, we only officially support the last two versions of Safari, so 14 and 15 currently. Honestly just working around Safari's recent bugs is hard enough, and that's where nearly all their users are. 

v1.9 exposed a bug in Safari 15 where textures randomly go black, and a workaround was added in v1.10.
